### DISCUSSION TITLE
How do I enable scroll bar?

### DISCUSSION THREAD

**Discussion by saiwing-yeung** [ghostty-org/ghostty#4228] [Q&A] [answered]
I looked in https://ghostty.org/docs/config/reference but didn't find anything relevant. Thank you.

**Comment by peterjc** [comment, marked as answer]
Not possible yet sadly, follow issue #111

**Comment by polarNerd204** [comment]
WOW can't use command f to find anything and cant add a freaking scroll slider to the right?!?!?!?  WTF ? WHERE do i sigh up to hear when these SIMPLE things have been added. So far I love it more than any other terminal em ive tried. But C'mon.

**Comment by jcollie** [comment]
Let's keep it professional and productive folks!
